### PR TITLE
Port To Python 3

### PR DESCRIPTION
**Pull request by Saumya-Mishra9129**
The Port to Python3 has done. The activity is tested it is running as expected. @quozl Kindly review it.

### CODE REVIEW

File: LocoSugarActivity.py [hunk 1]
```diff
@@ -1,3 +1,4 @@
+from builtins import str
```
Comment (quozl) [inline LocoSugarActivity.py]: Copyright must be before code.  Why is this import necessary?

File: game.py [hunk 2]
```diff
@@ -9,7 +9,11 @@
 # You should have received a copy of the GNU General Public License
 # along with this library; if not, write to the Free Software
 # Foundation, 51 Franklin Street, Suite 500 Boston, MA 02110-1335 USA
-
+from builtins import map
+from builtins import str
+from builtins import range
+from builtins import object
```
Comment (quozl) [inline game.py]: Why are these imports necessary?

File: game.py [hunk 3]
```diff
@@ -89,7 +93,7 @@ def __init__(self, canvas, parent=None, path=None):
 
         self._width = Gdk.Screen.width()
         self._height = Gdk.Screen.height()
-        self._scale = self._width / 1200.
```
Comment (quozl) [inline game.py:92]: Why use old_div when / should be sufficient?

**Review by quozl — CHANGES_REQUESTED** [review]
Thanks.  Where possible sprites.py should track other files by the same name in other activities; we don't want to multiply them without a need.

**Comment by ayushnawal** [comment]
Thanks for the pull request. Reviewed. Opened some issues after a quick review of the activity.

You can look at them after you finish the port and changes suggested by @quozl

File: LocoSugarActivity.py (hunk 1 above)
Comment (Saumya-Mishra9129) [reply LocoSugarActivity.py]: I have done porting by Futurize not by 2to3. Futurize automatically import some statements. These are un-necessary I guess. I will remove them.

File: game.py (hunk 3 above)
Comment (Saumya-Mishra9129) [reply game.py:92]: old_div in python 3 forces division operator to work just like it was working on python 2.  It doesnt impact code and will not imply any change in functionallity.

File: game.py (hunk 2 above)
Comment (Saumya-Mishra9129) [reply game.py]: These are un-necessary , I am removing.

**Comment by Saumya-Mishra9129** [comment]
I have made further commits in PR #13 .

**Comment by ayushnawal** [comment]
@Saumya-Mishra9129 I saw #13 along with this PR. It has become difficult to review now.

From a quick glance, as @chimosky pointed out I guess the mistake you did that you have created the branch `fixes` with `porting` as the parent branch.

You could have simply pushed the changes in this PR only with an appropriate commit message. If you want to fix the issue separately you should create another branch with master as its parent branch. Both review and merging for @quozl becomes difficult.

Commit messages can be made clearer, See [Making Commits](https://github.com/sugarlabs/sugar-docs/blob/master/src/contributing.md#making-commits)

In my opinion, all the commits of this PR are already in #13 , so this will be just an additional PR.

Else @quozl can suggest a better alternative for sure

**Comment by Saumya-Mishra9129** [comment]
@ayushnawal  As all the commits of this PR is already in #13 I should close this PR. Thanks for reviewing.

**Comment by chimosky** [comment]
@Saumya-Mishra9129 will you be closing this?

File: LocoSugarActivity.py (hunk 1 above)
Comment (quozl) [reply LocoSugarActivity.py]: Thanks.  I ask that porting is done using 2to3 unless there is a good reason for using Futurize.  At this stage, I don't know if you have a reason.

File: game.py (hunk 3 above)
Comment (quozl) [reply game.py:92]: It impacts reading by humans, and doesn't follow our conventions, so I'll expect it to go.

**Comment by Saumya-Mishra9129** [comment]
I am closing this one. Thanks for review.

File: game.py (hunk 3 above)
Comment (Saumya-Mishra9129) [reply game.py:92]: Okay Thanks.
